internal/config/git.go

```diff
@@ -92,3 +92,20 @@ func (c *Config) GetAll(name string) []string {
 	}
 	return res
 }
+func (c *Config) GetPrefix(prefix string) []string {
+	var processedMessages []string
+	for _, entry := range c.Entries {
+		if strings.HasPrefix(entry.Key, prefix) {
+			trimmedMsg := strings.TrimPrefix(entry.Key, prefix)
+			formattedArg := fmt.Sprintf(`strict="%s=%s"`, trimmedMsg, entry.Value)
```

Comment:
You would need to remove the `strict` from here if you use my other suggestion.

Personally, I don't think it makes sense to couple this particular method implementation to the `strict` argument; this method should only return the actual values with no formatting at all.

So, if your config has:

```
"receive.fsck.missingEmail", "ignore")
"receive.fsck.badTagName", "error")
```

I think this method should return

```
["missingEmail=ignore", "badTagName=error"]
```

And then, the formatting should happen at the callsite

Reply:
Agreed, thats the intended functionality.